src/download-center-config.ts

```diff
@@ -19,3 +21,24 @@ export interface DownloadCenterConfig {
   supported_browsers_link: string;
   tutorial_link: string;
 }
+export interface DownloadCenterConfigV2 {
+  previous_releases_link: string;
+  release_notes_link: string;
+  tutorial_link: string;
+  version: string;
```

Comment:
This feels odd – are multiple versions not supported in this variant?

Are they really fully distinct download center config schemas? I would have expected that the basic structure of the download center config would have stayed the same here, and we would just have applied the union type to `platform`, for example 

```js
    platform: ({
      arch: string;
      os: string;
      name: string;
    } & ({
      download_link: string;
    } | {
      packages: Package[];
    }))[];
```

Reply:
I agree with you, but I wrote the schema based on  https://github.com/mongodb/mongodb-atlas-cli/blob/main/tools/release/main.go#L58 which only supports one version. I'm still waiting for an answer from the Web Team for a formal json schema (or something) that I can use as an example.